### PR TITLE
Fixes autolayout error

### LINKED ISSUE #3056: Autolayout Constraint Error
#### Steps:
1. Launch WPiOS on an iPhone device
2. Open the Reader
3. Pick any post and tap over the comments section
4. Tap over the ReplyTextView
5. Rotate the device

#### Results:
The following error will be rendered onscreen:

```
Make a symbolic breakpoint at UIViewAlertForUnsatisfiableConstraints to catch this in the debugger.
The methods in the UIConstraintBasedLayoutDebugging category on UIView listed in <UIKit/UIView.h> may also be helpful.
2015-01-20 13:12:31.047 WordPress[5578:777068] Unable to simultaneously satisfy constraints.
	Probably at least one of the constraints in the following list is one you don't want. Try this: (1) look at each constraint and try to figure out which you don't expect; (2) find the code that added the unwanted constraint or constraints and fix it. (Note: If you're seeing NSAutoresizingMaskLayoutConstraints that you don't understand, refer to the documentation for the UIView property translatesAutoresizingMaskIntoConstraints) 
(
    "<NSLayoutConstraint:0x7fc9cb6eee30 V:|-(0)-[UIView:0x7fc9ce13eea0]   (Names: '|':UIView:0x7fc9ce1521f0 )>",
    "<NSLayoutConstraint:0x7fc9cb6f3ab0 V:[UIView:0x7fc9ce13eea0(54)]>",
    "<NSLayoutConstraint:0x7fc9cb6c9f50 V:[UIView:0x7fc9ce13eea0]-(0)-[UITableView:0x7fc9cbc1be00]>",
    "<NSLayoutConstraint:0x7fc9cb6ca9a0 V:[UITableView:0x7fc9cbc1be00]-(0)-[WordPress.ReplyTextView:0x7fc9cb6ceef0]>",
    "<NSLayoutConstraint:0x7fc9cb6d75a0 V:[WordPress.ReplyTextView:0x7fc9cb6ceef0]-(204)-|   (Names: '|':UIView:0x7fc9ce1521f0 )>",
    "<NSLayoutConstraint:0x7fc9ce22ab80 'UIView-Encapsulated-Layout-Height' V:[UIView:0x7fc9ce1521f0(207)]>"
)

Will attempt to recover by breaking constraint 
<NSLayoutConstraint:0x7fc9cb6ca9a0 V:[UITableView:0x7fc9cbc1be00]-(0)-[WordPress.ReplyTextView:0x7fc9cb6ceef0]>
```

### PR DESCRIPTION
**Pull request by jleandroperez**
Closes #3056 

/cc @aerych (Thank you sir!)

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m [hunk 1]
```diff
@@ -184,6 +185,27 @@ - (void)willRotateToInterfaceOrientation:(UIInterfaceOrientation)toInterfaceOrie
     [self.tableViewHandler refreshCachedRowHeightsForWidth:width];
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m]: According to the docs this API is 8.0 or later. Adding the declaration won't cause problems on iOS 7, but it won't be executed either.  Is this by design?

Comment (jleandroperez) [inline WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m]: Yup. (Strangely) the autolayout error doesn't even show up in iOS 7

Comment (aerych) [inline WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m]: Considering all the internal view hierarchy changes between 7 and 8, I'm not surprised at all. :p

**Comment by aerych** [comment]
Looks good @jleandroperez :shipit:

**Comment by jleandroperez** [comment]
Thank you Eric!

**Pull request merged**